docs/collecting-container-logs.md

```diff
@@ -214,8 +214,46 @@ Whereas log line 2 will be displayed as JSON:
 }
 ```
 
-> **Warning** Setting the format to `text` has certain consequences for multiline detection. See [here][troubleshooting_text_format] for
-> more details.
+##### Problem
```

Comment:
This is from #3247, right?

Reply:
Not at that time. I moved it here from fluentd documentation, because it is linked somewhere else (outside of fluentd documentation)